src/main/java/org/sonar/plugins/clojure/sensors/ancient/OutdatedDependency.java

```diff
@@ -0,0 +1,56 @@
+package org.sonar.plugins.clojure.sensors.ancient;
+
+import java.util.Objects;
+
+public class OutdatedDependency {
```

Comment:
In the beginning I thought we could use the `Issue.java` to represent any issue that the scanner would find. Now that you created a dedicated class for outdated dependencies, I am wondering if we should move Issue.java to the Eastwood package and make it Eastwood specific or change this class to use the Issue. What do you think?

Reply:
I think that moving issue to Eastwood issue is better because rest of the plugins also have spesific classes for their issues. This can of course be refactored later easily if necessary.

Reply:
Sounds good. I will do that.